### DISCUSSION TITLE
Feature Request: Disable Forwards

### DISCUSSION THREAD

**Discussion by neonknight** [modoboa/modoboa#1967] [Ideas]
Forwards are (in my opinion) the number one cause for email problems of all kinds. It would therefore be great if the option to define forwards could be disabled on a global, domain and per user base.
If forwarding is disabled, the UI option to define forwards should no longer be visible to logged in users. All previously defined forwards should be disabled.

**Comment by solkmaaker** [comment]
Agreed, unless SRS is used, there is no way in present day to forward e-mails to correctly configured destination servers that are using SPF, DKIM and DMARC verifications.
